Do not error attempting to kill a state-svc process that has already finished.

mitchell-as opened the pull request:
<!--do not remove this marker, its needed to replace info when ticket title is updated -->
<!--jira-description-action-hidden-marker-start-->

<table>
<td>
  <a href="https://activestatef.atlassian.net/browse/DX-1386" title="DX-1386" target="_blank"><img alt="Bug" src="https://activestatef.atlassian.net/rest/api/2/universal_avatar/view/type/issuetype/avatar/10303?size=medium" />DX-1386</a>  state update failed and has a double error message with the need for help part in the middle.
  </td></table>
  <br />
 

<!--jira-description-action-hidden-marker-end-->
<!--do not remove this marker, its needed to replace info when ticket title is updated -->

MDrakos reviewed (commented):
Won't this still show the `Something went wrong` header twice? Also, do we have any ideas as to why we cannot stop the service? That seems to be the issue described in the ticket.

mitchell-as commented:
Good catch. I don't see an easy way to eliminate the double header because the updater executes the installer and pipes its stdout and stderr directly to the terminal without having a way to read it or determine whether or not the err was a locale.InputError. I've reverted the tips fix and just addressed the state-svc killing failure. Let me know if you think there's a better way to address this ticket. It's a bit ambiguous as to what exactly the fix should be.

MDrakos reviewed (approved):
Nice! I think this covers it.
